apps/freelance/src/utils/select-options/options.ts

```diff
@@ -520,23 +524,23 @@ export const workExperience: SelectOptions[] = [
 		value: "5",
 	},
 	{
-		label: "more than 5 years",
+		label: "More than 5 years",
 		value: "more5",
 	},
 	{
-		label: "more than 10 years",
+		label: "More than 10 years",
 		value: "more10",
 	},
 	{
-		label: "more than 15 years",
+		label: "More than 15 years",
 		value: "more15",
 	},
 	{
-		label: "more than 20 years",
+		label: "More than 20 years",
 		value: "more20",
 	},
 	{
-		label: "more than 25 years",
+		label: "More than 25 years",
 		value: "more25",
 	},
 ];
```

Comment:
I dont know what is it. but all texts should be in translation file

Reply:
added  to translation file